build.gradle

```diff
@@ -823,7 +823,7 @@ task distDocker {
         dockerPlatform = "--platform ${project.getProperty('docker-platform')}"
         println "Building for platform ${project.getProperty('docker-platform')}"
       }
-      def gitDetails = getGitCommitDetails(7)
+      def gitDetails = getGitCommitDetails()
```

Comment:
8 seems fine, but if we are going to generate a sha that is different from the build artifact name, we may as well use all 40 here